src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  flex-grow: 0;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &:nth-child(-n+4) {
+    margin-bottom: 46px;
+  }
```

Comment:
Still, you add margins to BEM-block class. Make the same tag an element of higher-order block and put margins there (BEM mix)